blog/2024-02-27-deprecate-bind-hass-and-hass-components.md

```diff
@@ -0,0 +1,88 @@
+---
+author: Jan-Philipp Benecke
+authorURL: https://github.com/jpbede
+authorImageURL: https://avatars.githubusercontent.com/u/3989428?s=96&v=4
+title: "Deprecate use of @bind_hass and hass.components"
+---
+
+As of Home Assistant 2024.3, we deprecate the use of the `@bind_hass` decorator 
+and thus also the use of `hass.components`. A warning will be logged if used.
```

Comment:
I checked the core PR, and it will only handle the use of `bind_hass`. Ie, it won't handle the cases where we remove the `bind_hass` decorator. Users of those functions must adjust when we release the changes that where we've removed the decorator, otherwise their calls will blow up. I think we should clarify this.

Reply:
Added a phrase that we start to apply this to core now and custom integration authors need to adjust their code to avoid breakage.